src/flight_control/av_state.cpp

```diff
@@ -166,8 +169,44 @@ State AvState::fromArmed(DataDump dump)
                 return State::ARMED;
         }
     }
+    // If the safety checks (valves open, vents open, no pressure) are failed we go to the ERRORGROUND state
+    // TODO: ensure those are the right checks
+    if (!dump.valves.ValvesManual()|| (dump.prop.fuel_pressure <= 0 && dump.prop.LOX_pressure <= 0))
+    {
+        return State::ERRORGROUND;
+    }
+    // If the propulsion is OK we go to the READY state
+    // TODO: ensure those are the right checks
+    else if (dump.prop.fuel_pressure >= FUEL_PRESSURE_WANTED && dump.prop.LOX_pressure >= LOX_PRESSURE_WANTED) 
+    {
+        return State::READY;
+    }
+    return State::ARMED;
```

Comment:
These conditions overall need some correction. The switch over the `telemetry_cmd.id` seems deprecated.
The transition ARMED->READY is OK for the moment but will later be determined by the feedback from DPR boards (PR tanks pressure regulators).

Reply:
Yes, once we receive `PRESSURIZE` command from GS and we send `PRESSURIZE` to the DPRs, we wait for their `GO`